configs/milvus.yaml

```diff
@@ -285,7 +285,7 @@ indexNode:
 
 dataCoord:
   channel:
-    watchTimeoutInterval: 120 # Timeout on watching channels (in seconds). Datanode tickler update watch progress will reset timeout timer.
+    watchTimeoutInterval: 300 # Timeout on watching channels (in seconds). Datanode tickler update watch progress will reset timeout timer.
```

Comment:
What is the reason for escalating the value of the `watchTimeoutInterval` parameter?

Reply:
Because many bugs are caused by reassign channels without cleanup it.
1. recovery may failed forever it watch can not succeed in 120s
2. release and watch failed may cause double assign.
Change to 300s may not be safe, but it covered more general cased